### PR TITLE
Portfolio mobile menu is 

### PR DESCRIPTION
**Pull request by misikirdelelew**
In this PR we have done
Javascript DOM manipulation for mobile menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,15 +18,16 @@
...
         <nav class="menu">
             <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+            <i class="fa fa-close" id="close"></i>
             <ul class="desktop-menu">
-                <li><a href="" class="desktop-menu-item" id="Porfolio">Portfolio</a></li>
-                <li><a href="" class="desktop-menu-item" id="About">About</a></li>
-                <li><a href="" class="desktop-menu-item" id="Contact">Contact</a></li>
+                <li><a href="#headlinesec" class="desktop-menu-item" id="portfolio">Portfolio</a></li>
+                <li><a href="#aboutsection" class="desktop-menu-item" id="about">About</a></li>
+                <li><a href="#contactsection" class="desktop-menu-item" id="contact">Contact</a></li>
```
Comment (Nemwel-Boniface) [inline index.html:25]: - [ ] The shade of blue color that is used in your project is of a lighter shade than the one that is specified in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546&t=n1NHJL9g7qXZac5C-0) design. Kindly ensure that you use the correct shade of the background color to meet all the project requirements.

**This is the color of your background when I check it out**

image

**This is the desired background color design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @MishDM       :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far ! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:25]: well resolved! ✔️

File: index.html [hunk 2]
```diff
@@ -17,15 +18,16 @@
         </div>
         <nav class="menu">
             <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+            <i class="fa fa-close" id="close"></i>
```
Comment (rloterh) [inline index.html:21]: - [ ] OPTIONAL
Nice job with the hamburger, however, I we can further improve upon the **user experience** by ensuring that the _close button_ is behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _close button_ on _mouse hover_ events.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Misikir,  

Your project is complete!🏁 Good job! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- Mobile menu works as intended!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- I suggest one _Authors_ section should do for all contributor details. Let's endeavor to improve upon this aspect:
image

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**